src/plugins/analysis/ipc/docker/ipc_analyzer/ipc_analyzer.py

```diff
@@ -336,6 +336,38 @@ def find_function_ref_strings(function_name):
     return strings
 
 
+def find_function_constants(function_name):
+    """
+    Get all constants that are used as operands in the function with name `function_name`.
+
+    :param function_name: The name of the function.
+    :type function_name: str
+    :return: a list of int/long constants referenced in the function as strings
+    :rtype: list[str]
+    """
+    try:
+        function = getGlobalFunctions(function_name)[0]
+    except (IndexError, TypeError):
+        print("Error: Function {} not found.".format(function_name))
+        return []
+
+    constants = []
+    if function is not None:
+        body = function.getBody()
+        instruction_iterator = currentProgram.getListing().getInstructions(body, True)
+
+        for instruction in instruction_iterator:
+            for i in range(instruction.getNumOperands()):
+                for operand in instruction.getOpObjects(i):
+                    try:
+                        value = operand.getValue()
+                    except AttributeError:
+                        continue
+                    if value is not None and isinstance(value, (int, long)):
+                        constants.append(str(value))
+    return constants
```

Comment:
Nit: I personally prefer less indentation, which in this case can be archived by adding:
```python
if function is None:
    return []
```

Also: Why can function be none if we did not get an exception in the call to `getGlobalFunctions`?!
I think it [cannot](https://ghidra.re/ghidra_docs/api/ghidra/program/flatapi/FlatProgramAPI.html#getGlobalFunctions(java.lang.String)).